packages/server/src/handlers/__tests__/check.spec.ts

```diff
@@ -104,3 +104,16 @@ test("handleCheckWord: Valid, Unused Letter", () => {
     letterState: [0, 2, 2, 2, 2],
   });
 });
+
+test("handleCheckWord: Valid, Unused Letter", () => {
```

Comment:
Another test case would be useful here to confirm the first "correct letter but in the wrong spot" doesn't remain yellow. Perhaps `CORRECT_WORD: "CRANE"` and `maybeWord: "ERASE"`?